src/js/msp/MSPHelper.js

```diff
@@ -1070,6 +1070,7 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 FC.GOVERNOR.gov_rpm_filter                   = data.readU8();
                 FC.GOVERNOR.gov_tta_filter                   = data.readU8();
                 FC.GOVERNOR.gov_ff_filter                    = data.readU8();
+                FC.GOVERNOR.gov_min_spoolup_throttle         = data.readU8();
```

Comment:
The MSP changes should be enclosed in an MSP version check, as these will be are new in Rotorflight 2.2

Reply:
Done